File: fix_guru.py
import re

with open('src/pages/KasihIbuGuru.tsx', 'r') as f:
    code = f.read()

code = code.replace('}</>', '')
code = code.replace('      )}\n      )}', '      )}')
code = code.replace('{/* Modal Tukar Poin', ')}\n      {/* Modal Tukar Poin')

with open('src/pages/KasihIbuGuru.tsx', 'w') as f:
    f.write(code)

File: fix_guru.js
const fs = require('fs');
let code = fs.readFileSync('src/pages/KasihIbuGuru.tsx', 'utf-8');

code = code.replace(/}<\/>/g, '');
code = code.replace(/{\/\* Modal Tukar Poin \*\//, ')} {/* Modal Tukar Poin */}');

fs.writeFileSync('src/pages/KasihIbuGuru.tsx', code);
